playbooks/debug_box.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: all
+  become: true
+  roles:
+    - sos_report
+
+- hosts: all
+  become: true
```

Comment:
You don't necessarily need to split these into two plays in one file, you can have:

```
roles:
  -
tasks:
  -
```